src/main/java/software/amazon/event/kafkaconnector/mapping/DefaultTimeMapper.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package software.amazon.event.kafkaconnector.mapping;
+
+import java.time.Instant;
+import org.apache.kafka.connect.sink.SinkRecord;
+import software.amazon.event.kafkaconnector.EventBridgeSinkConfig;
+
+public class DefaultTimeMapper implements TimeMapper {
+
+  @Override
+  public Instant getTime(SinkRecord sinkRecord) {
+    return null;
```

Comment:
Please add a short comment that null will leverage the time of PutEvents with a refernce to: https://docs.aws.amazon.com/eventbridge/latest/APIReference/API_PutEventsRequestEntry.html

Or for readability we might leverage a constant to explain the [magic value / number](https://en.wikipedia.org/wiki/Magic_number_(programming)):

`private static final Instant PUT_EVENTS_DEFAULT_TIME = null;`

Reply:
Good point, added a documentation reference.